src/tls.c

```diff
@@ -10923,7 +10926,10 @@ int TLSX_EarlyData_Use(WOLFSSL* ssl, word32 maxSz, int is_response)
     }
 
     extension->resp = is_response;
-    extension->val  = maxSz;
+    /* In QUIC, earlydata size is either 0 or 0xffffffff.
+     * Override any size between, possibly left from our intial value */
+    extension->val  = (WOLFSSL_IS_QUIC(ssl) && is_response
```

Comment:
Some extra '(' and ')' would help readability here. "(is_response && maxSz > 0)" instead of "is_response && maxSz > 0"

Reply:
reformatted to have the condition on one line. extra parenthesis seem confusing since the two `&&` are on the same level.